StopSelector/index.html

```diff
@@ -15,6 +15,19 @@
   <script src="js/underscore.min.js"></script>
   <script src="js/main.js"></script>
 </head>
+  <style>
+    h2 {
+      display: inline-block;
+      font-size: 1.5em;
+      margin-before: 0.83em;
+      margin-after: 0.83em;
+      margin-start: 0;
+      margin-end: 0;
+      font-weight: bold; }
+    span {
+      color: "white";
+    }
+  </style>
```

Comment:
No inline style, please.  This should go in [the CSS file](https://github.com/umts/BusInfoBoard/blob/master/StopSelector/css/main.css)

Having said that, if you're going to use `<style>` tags, they go inside the `<head>`